app/search/-components/filter-facet/template.hbs

```diff
@@ -0,0 +1,106 @@
+<div
+    data-test-filter-facet
+    local-class='facet-wrapper'
+>
+    {{#let (unique-id @propertyRecord.label) as |facetElementId|}}
+        <Button
+            data-analytics-name='filter facet toggle {{@propertyRecord.label}}'
+            data-test-filter-facet-toggle={{@propertyRecord.label}}
+            @layout='fake-link'
+            local-class='facet-expand-button'
+            aria-controls={{facetElementId}}
+            aria-expanded={{not this.collapsed}}
+            {{on 'click' (fn this.toggleFacet this.label)}}
+        >
+            <span>{{@propertyRecord.label}}</span>
+            <FaIcon @icon={{if this.collapsed 'caret-down' 'caret-up'}} />
+        </Button>
+        {{#if this.fetchFacetValues.isRunning}}
+            <LoadingIndicator @dark={{true}} />
+        {{else if this.fetchFacetValues.isError}}
```

Comment:
I don't think I knew we could do `else if` in templates. Though it looks like we've used it 51 times before, so surely I've come across it.